clock_exercise/clockmanager.h

```diff
@@ -0,0 +1,41 @@
+#ifndef CLOCKMANAGER_H
```

Comment:
Personalmente ti consiglio di usare `#pragma once` al posto delle include guards fatte con `#ifdef`.
Così facendo corri un rischio inferiore di fare confusione, magari per dei copia-incolla o per un typo, in più `#pragma once` anche se non è standard è comunque supportato da tutti i compilatori ed è anche ottimizzato rispetto alle include guards.